cognite/client/data_classes/datapoints_subscriptions.py

```diff
@@ -369,17 +371,17 @@ def _metadata(key: str) -> list[str]:
     return ["metadata", key]
 
 
-class DatapointSubscriptionFilterProperties:
-    description = ["description"]
-    external_id = ["externalId"]
+class DatapointSubscriptionFilterProperties(EnumProperty):
+    description = auto()
+    external_id = auto()
     metadata = _metadata
```

Comment:
I think this breaks filtering on metadata properties. Since this is an EnumProperty, a few helpers will invoke `.as_reference()` on it, which will result in `[["metadata", "<key>"]]` now rather than `["metadata", "<key>"]` when using `DatapointSubscriptionFilterProperties.metadata("<key>")`

Reply:
No, it's just a string, so it will pass through as expected (I've tested)